src/identity/apis/createOrg.ts

```diff
@@ -0,0 +1,67 @@
+// API Calls
+import {
+  getClusters,
+  postOrg,
+  OrganizationCreateRequest,
+} from 'src/client/unityRoutes'
+
+// Error Types
+import {
+  ServerError,
+  UnauthorizedError,
+  UnprocessableEntityError,
+} from 'src/identity/apis/auth'
+
+export class OrgNameConflictError extends Error {
+  constructor(message) {
+    super(message)
+    this.name = 'OrgNameConflictError'
+  }
+}
+
+export const createNewOrg = async (
+  organizationCreateRequest: OrganizationCreateRequest
+) => {
+  const {orgName, provider, region} = organizationCreateRequest
+
+  const response = await postOrg({
+    data: {
+      orgName,
+      provider,
+      region,
+    },
+  })
+
+  if (response.status === 401) {
+    throw new UnauthorizedError(response.data.message)
+  }
+
+  if (response.status === 409) {
+    throw new OrgNameConflictError(response.data.message)
+  }
+
+  if (response.status === 422) {
+    throw new UnprocessableEntityError(response.data.message)
+  }
+
+  if (response.status === 500) {
+    throw new ServerError(response.data.message)
+  }
+
+  // Status code in response when successfully creating an org is 201.
+  return response.data
+}
+
+export const fetchClusterList = async () => {
+  const response = await getClusters({})
+
+  if (response.status === 401) {
+    throw new UnauthorizedError(response.data.message)
+  }
+
+  if (response.status === 500) {
+    throw new ServerError(response.data.message)
+  }
+
+  return response.data
+}
```

Comment:
is this where all the org api functionality will be? if so, maybe just call this file `src/identity/apis/org.ts`

Reply:
There are other org related functions in the identity api file (auth.ts).  But at this point that file has grown oversize with everything included in it. 

You'll see a bigger diff here shortly to reflect the fact that I'm going to split auth.ts up into three separate files, plus a separate file for the types.